test/src/test/java/org/corfudb/logreplication/runtime/NegotiationResponseTest.java

```diff
@@ -0,0 +1,198 @@
+package org.corfudb.logreplication.runtime;
+
+import org.corfudb.infrastructure.LogReplicationRuntimeParameters;
+import org.corfudb.infrastructure.logreplication.receive.LogReplicationMetadataManager;
+import org.corfudb.logreplication.fsm.LogReplicationEvent;
+import org.corfudb.logreplication.infrastructure.LogReplicationNegotiationException;
+import org.corfudb.protocols.wireprotocol.logreplication.LogReplicationNegotiationResponse;
+import org.corfudb.runtime.CorfuRuntime;
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+import static org.mockito.Mockito.RETURNS_DEEP_STUBS;
+import static org.mockito.Mockito.doReturn;
+import static org.mockito.Mockito.mock;
+import static org.mockito.BDDMockito.given;
+
+@SuppressWarnings("checkstyle:magicnumber")
```

Comment:
Can you please add a general description to this suite. What we are "mocking", what is tested, unit? IT?